examples/opt_serving/model/opt_model.py

```diff
@@ -529,25 +529,28 @@ def init_model_aval(config):
     return model, params
 
 
-def init_cache_aval(config, batch_size):
+def init_cache_aval(config):
```

Comment:
revert all changes of `init_cache_aval`. Let us keep `batch_size` as an argument because we may use multiple batch sizes for the same model config

Reply:
Then shall we remove `batch_size` in `config`? Since we have a `batch_size` outside the config, it is weird to have another.